### PR TITLE
Sveta_Assignment_20

### PR DESCRIPTION
**Pull request by svetakanani**
### Evidence :file_folder:

image

### Checklist before requesting a review :ballot_box_with_check:

- [x] I have performed a self-review of my code
- [x] My changes generate no compile time errors

### CODE REVIEW

File: src/sveta/Assignment_20_EvenNo.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+				count++;
+			}
+			i++;
+		}
+	}
+	
+	public void forprint() {
+		System.out.println("using for :");
+		int count=1;
+		for (int i=2;count<=5;i++) {
```
Comment (harshvegada) [inline src/sveta/Assignment_20_EvenNo.java:21]: We have to print first 5 number,not to restrict it to 5 iteration

**Review by harshvegada — APPROVED** [review]
Logic looks good to me

File: src/sveta/Assignment_20_EvenNo.java (hunk 1 above)
Comment (svetakanani) [reply src/sveta/Assignment_20_EvenNo.java:21]: sorry I resolve it. but In this case my intention is restrict the value that was print five time.
I mean print 5 even no.

**Comment by MargiVegada** [comment]
Add assignment description too

**Pull request merged**